### ISSUE TITLE
Hardcoded custom field IDs

### ISSUE THREAD

**Issue by pbnj** [nortonlifelock/jira#3]
We're running into issues with `aegis` running against our JIRA instance because of [these hardcoded custom field IDs](https://github.com/nortonlifelock/jira/blob/5ed2c9231b2ace2a949766e839b5e5d0b6feddab/json_transition.go#L34-L35).

**Comment by ryan-everhart** [comment]
Good catch! I've accounted for those fields in the JIRA ISSUE3 branch. It will need to go through more testing before merging with the master branch, but if you want to try it locally you can run `go get github.com/nortonlifelock/jira@ISSUE3` in your Aegis directory
